CONTRIBUTING.md

```diff
@@ -158,9 +158,10 @@ Harbor backend is written in [Go](http://golang.org/). If you don't have a Harbo
 |   1.9    |    1.12.12    |
 |   1.10   |    1.12.12    |
 |   2.0    |    1.13.15    |
-|   2.1    |    1.14.13     |
+|   2.1    |    1.14.13    |
```

Comment:
Not strictly relevant to this PR, but have we mentioned that notary components are compiled using a different version of Go?

Reply:
will raise another PR